modules/effects/spec/actions.spec.ts

```diff
@@ -58,10 +66,10 @@ describe('Actions', function() {
     actions.forEach(action => dispatcher.next(action));
   });
 
-  it('should let you filter out actions', function() {
-    const actions = [ADD, ADD, SUBTRACT, ADD, SUBTRACT];
-    const expected = actions.filter(type => type === ADD);
+  const actions = [ADD, ADD, SUBTRACT, ADD, SUBTRACT];
+  const expected = actions.filter(type => type === ADD);
 
+  it('should let you filter out actions', function() {
     actions$
       .pipe(
         ofType(ADD),
```

Comment:
TS is complaining about the next line `map(update => update.type)`
This can be solved if we define our actions as `let actions$: Actions<AddAcction | SubtractAction>;`

Reply:
Sorry, I haven't even been able to run the typechecker on the whole PR, because of the other deps. Fixed.

Reply:
CI is green. I assume this means it is not longer an issue.

Reply:
LGTM 👍